### PR TITLE
Fix crash in <CephFs />

### LINKED ISSUE #5480: Cant open pod info with attached Cephfs volume
**Describe the bug**
If pod have attached volume CephFS(like this: https://github.com/kubernetes/examples/blob/master/volumes/cephfs/cephfs-with-secret.yaml), i cant open pod info about this pod into Lens(get error).

**To Reproduce**
1. Create pod with CephFS volume
2. Go to k8s cluster -> Workloads-> Pods
3. Click on created pod in pods list

**Expected behavior**
I expected to see pod info

**Screenshots**
[If applicable, add screenshots to help explain your problem.](https://i.postimg.cc/VN5HWv6F/lens-pod-cephfs-error.png)

**Environment (please complete the following information):**
- Lens Version: 5.5.0-latest.20220525.1
- OS: Windows 10 21H2(Build 19044.1706)
- Installation method (e.g. snap or AppImage in Linux): download and install exe file

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
Component stack:


    at CephFs (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:535980:20)
    at VolumeVariant (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:536605:20)
    at div
    at observerComponent (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:266183:16)
    at div
    at PodDetails (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:536682:9)
    at div
    at div
    at div
    at Animate (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:573737:9)
    at NonInjectedDrawer (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:579968:9)
    at KubeObjectDetails (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:584458:9)
    at ErrorBoundary (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:582139:9)
    at NonInjectedClusterFrame (http://23710c36917751940dd34314cbc53496.localhost:58153/build/chunks/888.js:2924:9)
    at ThemeProvider (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:127160:24)
    at Router (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:454676:30)
```
```
Error stack:

TypeError: Cannot read properties of undefined (reading 'toString')
    at CephFs (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:535988:83)
    at Ch (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:453189:137)
    at ck (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:453299:460)
    at bk (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:453282:347)
    at ak (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:453282:278)
    at Tj (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:453282:138)
    at Lj (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:453275:163)
    at http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:453155:115
    at exports.unstable_runWithPriority (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:467222:343)
    at gg (http://23710c36917751940dd34314cbc53496.localhost:58153/build/Lens.js?266aed5a965262b158f7:453154:325)
```

**Additional context**
Error only with pods where attached CephFS volume.

### PR DESCRIPTION
**Pull request by Nokel81**
- Add some basic unit tests

Signed-off-by: Sebastian Malton <sebastian@malton.name>

fix #5480

### CODE REVIEW

File: src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const CephFs: VolumeVariantComponent<"cephfs"> = (
             </DrawerItem>
           )
       }
-      <DrawerItem name="Readonly">
+      <DrawerItem name="Readonly" valueTestId="cephfs-readonly">
```
Comment (ixrock) [inline src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx]: Why to use extra `props.valueTestId` when we can pass directly from `<Drawer data-testid="some-value"/>` to `<span>`?

**Review by jim-docker — DISMISSED** [review]
couldn't reproduce the issue on 5.5.1, but no crash on this branch either

File: src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx (hunk 1 above)
Comment (Nokel81) [reply src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx]: I don't see where we can pass in `data-testid` on `Drawer`? That only works on the base HTML react elements. Plus we aren't producing a `Drawer` anyway.

Comment (ixrock) [reply src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx]: Nope, it works on all elements and react components. You should pass it on `DrawerItem` not `Drawer` _(wrong example above)_

Comment (ixrock) [reply src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx:42]: Nope, it works on all elements and react components. You should pass it on `DrawerItem` not `Drawer` _(wrong example above)_

Comment (ixrock) [reply src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx]: For auto-complete you can declare in `DrawerItem`'s props, but it's not required.

Comment (aleksfront) [reply src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx]: I agree that unusual `valueTestId` should be avoided.

**Review by aleksfront — DISMISSED** [review]
Minor comment left.

File: src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx (hunk 1 above)
Comment (Nokel81) [reply src/renderer/components/+workloads-pods/details/volumes/variants/ceph-fs.tsx]: Oh thanks for the info. I have now done that.

**Review by aleksfront — APPROVED** [review]

**Review by ixrock — APPROVED** [review]
LGTM

**Pull request merged**